charts/gxf/Chart.yaml

```diff
@@ -1,15 +1,15 @@
 apiVersion: v2
 name: gxf
 description: Generic GXF Helm chart
-version: '1.2.3'
+version: '1.2.4'
 icon: https://artwork.lfenergy.org/projects/grid-exchange-fabric/abbrev/color/grid-exchange-fabric-abbrev-color.png
 maintainers:
   - name: OSGP
     email: gxf@lists.lfenergy.org
     url: https://www.lfenergy.org/projects/gxf/
 dependencies:
 - name: activemq-artemis
-  version: "0.0.5"
+  version: "0.0.6"
```

Comment:
Shouldn't this be version 0.0.7 like in the chart.yaml of activeMQ?

Reply:
The build will fail: 998f4a8
Version 0.0.7 does not exist in the repository yet. So this should be changed in a next iteration.